tailwind.config.ts

```diff
@@ -3,6 +3,99 @@ import type { Config } from "tailwindcss";
 export default {
   content: ["./app/**/{**,.client,.server}/**/*.{js,jsx,ts,tsx}"],
   theme: {
+    colors:{
+      "gray":{
+        99: "hsla(250, 10%, 99%, 1)",
```

Comment:
Should be in the hundreds, etc "bg-slate-100" instead of "bg-slate-10", as it is by default

Reply:
hmm do you also want the numbering to go by shade instead of tint (i.e. dark (900) to light (100) instead of dark (100) to light (900) or similar)

Reply:
yes